spec/spec_helper_local.rb

```diff
@@ -29,15 +29,53 @@
   end
 end
 
-def with_debian_facts
+shared_context 'ArchLinux' do
   let :facts do
     {
       kernel: 'Linux',
       os: {
-        name: 'Debian',
-        release: { full: '8.0' },
+        name: 'ArchLinux',
```

Comment:
why do you mock the facts here for arch linux? facterdb has a factset for it

Reply:
updating to:
```
shared_context 'ArchLinux' do
  let(:facts) { on_supported_os['archlinux-x86_64'] }
end
```

Reply:
Ok, this doesn't seem to work

Reply:
Including archlinux as I linked above does not seem to work.
Looking into facterdb it seems that archlinux is no longer included in the newest facts, i.e. 4.3 forward
https://github.com/voxpupuli/facterdb/tree/master/facts/4.3